code/modules/client/preference_setup/loadout/items/suit.dm

```diff
@@ -291,6 +291,8 @@
 	dominiacape["white dominian cape, caladius"] = /obj/item/clothing/accessory/poncho/dominia_cape/caladius/white
 	dominiacape["dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao
 	dominiacape["white dominian cape, zhao"] = /obj/item/clothing/accessory/poncho/dominia_cape/zhao/white
+	dominiacape["dominian cape, military"]	= /obj/item/clothing/accessory/poncho/dominia_cape/mantle
+	dominiacape["dominian cape, medical"] = /obj/item/clothing/accessory/poncho/dominia_cape/hospital
```

Comment:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/a605f5e5-3533-4a36-9e92-14dd4f4ff445)
this already exists in the loadout